app/src/ui/about/about.tsx

```diff
@@ -232,7 +237,7 @@ export class About extends React.Component<IAboutProps, IAboutState> {
           </Row>
           <h2>{name}</h2>
           <p className='no-padding'>
-            Version {version} ({releaseNotesLink})
+            Version <span title='Click to copy' className='version-text' onClick={this.onClickVersion}>{version} <Octicon symbol={OcticonSymbol.clippy} /></span> ({releaseNotesLink})
```

Comment:
In the mean time, I think simply making the text selectable would be an A+ improvement.